lib/ModuleInfo.js

```diff
@@ -81,12 +81,40 @@ const ModuleInfoPlatform = {
 
 const HEADER_SIZE = 24;
 
+function moduleFunctionToChar(f) {
```

Comment:
Nitpicking, but all these functions should potentially exist outside of this particular source file with just the definitions here for simplicity.

Reply:
Yes, good call. I will leave it for now, but I'd like to do a refactor of this project and rework as ES6, and add documentation. It's not always entirely obvious how things work, especially when we have 3 different structures the information is represented in.